event-handlers/src/main/java/no/sikt/nva/nvi/events/evaluator/calculator/CreatorVerificationUtil.java

```diff
@@ -52,18 +54,37 @@ public CreatorVerificationUtil(AuthorizedBackendUriRetriever authorizedBackendUr
         this.organizationRetriever = new OrganizationRetriever(uriRetriever);
     }
 
-    public List<VerifiedNviCreator> getVerifiedCreatorsWithNviInstitutionsIfExists(JsonNode publication) {
-        var verifiedCreatorsWithNviInstitutions = getVerifiedCreatorsWithNviInstitutions(publication);
+    public List<VerifiedNviCreator> getVerifiedCreatorsWithNviInstitutions(JsonNode body) {
+        return getJsonNodeStream(body, JSON_PTR_CONTRIBUTOR)
+                   .filter(CreatorVerificationUtil::isVerified)
+                   .filter(CreatorVerificationUtil::isCreator)
+                   .map(this::toVerifiedNviCreator)
+                   .filter(CreatorVerificationUtil::isAffiliatedWithNviOrganization)
+                   .toList();
+    }
+
+    public List<UnverifiedNviCreator> getUnverifiedCreatorsWithNviInstitutions(JsonNode body) {
+        return getJsonNodeStream(body, JSON_PTR_CONTRIBUTOR)
+                   .filter(x -> !isVerified(x))
```

Comment:
What is `x`? Rename to `contributorNode`?

Reply:
Also, these methods have several lines in common - extract?

Reply:
What could we extract?

Reply:
Took a closer look - not much 😄  Resolved